ideas/87-new-protocol.md

```diff
@@ -63,26 +63,56 @@ Reasons for that are following:
 - Contributor: @cammellos
 - PM: @chadyj
 
-### Requirements & Dependencies
+
+## Current status of the Swarm
+
+- Weekly sync call on Fridays (contact eric@status.im if you want to join)
+- Focus on solving issues, particularly this one https://github.com/status-im/status-react/issues/4034
```

Comment:
This status looks like the mandate of #99.  If both are working on the same issue then would it be worthwhile to merge efforts?